webapp/src/components/left_sidebar_header/left_sidebar_header.jsx

```diff
@@ -25,7 +26,25 @@ export default class LeftSidebarHeader extends React.PureComponent {
                     className='icon fa fa-plug'
                     style={iconStyle}
                 />
-                {'Demo Plugin: '} {this.props.enabled ? <span>{ 'Enabled' }</span> : <span>{ 'Disabled' }</span>}
+                <FormattedMessage
+                    id='sidebar.demo'
+                    defaultMessage='Demo Plugin:'
+                />
+                {' '}
+                {this.props.enabled ?
+                    <span>
+                        <FormattedMessage
+                            id='sidebar.enabled'
+                            defaultMessage='Enabled'
```

Comment:
This battle was fought long ago, defaultMessage is the way it is. It provides a default even if translations are missing, `en.json` is for the translations server. I don't mind now because the automation takes care of creating `en.json`.

Reply:
Thanks for the explanation!